dibs/inference/svgd.py

```diff
@@ -57,7 +57,7 @@ class MarginalDiBS(DiBS):
 
     def __init__(self, *,
                  x,
-                 inference_model,
+                 graph_model, likelihood_model,
```

Comment:
This change is missing in `JointDiBS`. I'm pushing a change that fixes this now to this branch also adapting some of the docstrings. Make sure you try running the code when making future changes and adapt all docstrings according to the changes